File: TP2-Python/TP2-Kode Program/televisi.py

from elektronik import Elektronik

class Televisi(Elektronik):
    def __init__(self, id_produk, asal_produk, harga, lokasi_penyimpanan, nama, merk, ukuran, stok, tahun_produksi):
        super().__init__(id_produk, asal_produk, harga, lokasi_penyimpanan, nama, merk)
        self._ukuran = ukuran
        self._stok = stok
        self._tahun = tahun_produksi

    # Getter methods
    def get_ukuran(self):
        return self._ukuran

    def get_stok(self):
        return self._stok
    
    def get_tahun(self):
        return self._tahun

  

File: TP2-Java/Program Java/Televisi.java
public class Televisi extends Elektronik {
    protected String ukuran;
    protected int stok;
    protected int tahun_produksi;

    public Televisi(String id_produk, String asal_produk, double harga, String lokasi_penyimpanan, String nama, String merk, String ukuran, int stok, int tahun_produksi) {
        super(id_produk, asal_produk, harga, lokasi_penyimpanan, nama, merk);
        this.ukuran = ukuran;
        this.stok = stok;
        this.tahun_produksi = tahun_produksi;
    }

    // Getter methods
    public String getUkuran() {
        return ukuran;
    }

    public int getStok() {
        return stok;
    }

     public int getTahunProduksi() {
        return tahun_produksi;
    }

}
